pyproject.toml

```diff
@@ -11,14 +11,16 @@ keywords = ["poetry", "bump", "version", "plugin"]
 
 [tool.poetry.dependencies]
 python = "^3.7"
-poetry = "^1.2.0a1"
+poetry = "^1.2.1"
```

Comment:
Poetry version should not change.

Reply:
I will change it back.

Reply:
Is poetry.lock up to date now? Did you run `poetry lock --no-update`?

Reply:
Oh actually, I take it back. Do not merge.
For some reason I cannot get this to work with `poetry 1.2.0a1`. During the resolving dependency step, I just get an error like this:

![image](https://user-images.githubusercontent.com/38586420/191709372-d62cdcf2-f266-40fd-9eb3-c99d8a83b252.png)

I am not sure why it cannot set attribute. I need to look into this.
Also I was wondering, is there any reason why we cannot use `poetry 1.2.1` ?

Reply:
People using poetry 1.2.0a1 might also want to use this plugin.

Reply:
Can you get it working with `^1.2.0a2`? `1.2.0a1` might be [broken](https://github.com/python-poetry/poetry/issues/4351).

Reply:
Ahh sorry. You make sense. 
It is interesting that you should say that because I am having issues with `1.2.0a1`.
I will try `1.2.0a2` and see how it goes.

Reply:
It is all working for `poetry 1.2.0a2`.
I have updated `poetry.lock` and `pyproject.toml`.